cmd/analyze/analyze.go

```diff
@@ -69,14 +74,36 @@ var AnalyzeCmd = &cobra.Command{
 				os.Exit(1)
 			}
 		}
-
 		// print results
 		output, err := config.PrintOutput(output)
 		if err != nil {
 			color.Red("Error: %v", err)
 			os.Exit(1)
 		}
 		fmt.Println(string(output))
+
+		// Interactive mode
+		if interactiveMode && explain {
```

Comment:
Shouldnt we disallow the interactive mode with json output ?

Reply:
Good idea, less tokens